src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,7 +59,7 @@ public PersonCard(Person person, int displayedIndex) {
         name.setText(person.getName().value);
         phone.setText(person.getPhone().value);
         sex.setText(getFullSexString(person.getSex().value));
-        employmentType.setText(getFullEmploymentTypeString(person.getEmploymentType().value));
```

Comment:
hmm, can change? to .value? follow the standard from getName and getPhone

Reply:
As mentioned, value of PayRate is a double, hence need to use toString method